### PR TITLE
Remodel EACCESS section to be persistant.

### LINKED ISSUE #32: Linux FAQ "Failed to initialize virtual device" EACCESS section is not persistent
The suggested fix is to make `/dev/uinput` user-writeable. This is not necessary when udev rules are installed, which is necessary for proper tablet connection anyway.

Instead, the fix should state something along the lines of ensuring udev rules are installed, and then running `sudo udevadm control --reload-rules && sudo udevadm trigger`, and then rebooting if that does not fix it. If that still doesn't fix it they have broken udev rules.

### PR DESCRIPTION
**Pull request by jamesbt365**
fixes #32 

This is ugly and horribly formatted (much like my face), I will cleanup this pr when I have time after i wake up.

### CODE REVIEW

File: OpenTabletDriver.Web/Views/Wiki/FAQ/Linux.cshtml [hunk 1]
```diff
@@ -106,9 +106,17 @@
...
-        sudo chmod 0666 /dev/uinput
+    <p>This occurs usually when the udev rules are not correct, or haven't been reloaded.
+    Make sure your rules are correct by heading <a href="#fail-device-streams">here</a>.
+    </p>
+    <codeblock class="mt-2" language="sh">
+    sudo udevadm control --reload-rules
+    # Skips the step requiring a replug of the tablet.
+    udevadm trigger
+    # Helps ensure that OpenTabletDriver uses the tablet correctly.
+    systemctl --user restart opentabletdriver
```
Comment (InfinityGhost) [inline OpenTabletDriver.Web/Views/Wiki/FAQ/Linux.cshtml:117]: Fix indentation

File: OpenTabletDriver.Web/Views/Wiki/FAQ/Linux.cshtml [hunk 2]
```diff
@@ -106,9 +106,17 @@
 </h3>
 <div class="ms-3">
     <h4>EACCESS</h4>
-    <codeblock class="mt-2">
-        sudo chmod 0666 /dev/uinput
+    <p>This occurs usually when the udev rules are not correct, or haven't been reloaded.
+    Make sure your rules are correct by heading <a href="#fail-device-streams">here</a>.
+    </p>
```
Comment (InfinityGhost) [inline OpenTabletDriver.Web/Views/Wiki/FAQ/Linux.cshtml:111]: Fix formatting

**Review by InfinityGhost — CHANGES_REQUESTED** [review]

File: OpenTabletDriver.Web/Views/Wiki/FAQ/Linux.cshtml (hunk 2 above)
Comment (vedattt) [reply OpenTabletDriver.Web/Views/Wiki/FAQ/Linux.cshtml:111]:
```suggestion
    <p>
        This occurs usually when the udev rules are not correct, or haven't been reloaded.
        Make sure your rules are correct by heading <a href="#fail-device-streams">here</a>.
    </p>
```

File: OpenTabletDriver.Web/Views/Wiki/FAQ/Linux.cshtml (hunk 1 above)
Comment (vedattt) [reply OpenTabletDriver.Web/Views/Wiki/FAQ/Linux.cshtml:117]:
```suggestion
        sudo udevadm control --reload-rules
        # Skips the step requiring a replug of the tablet.
        udevadm trigger
        # Helps ensure that OpenTabletDriver uses the tablet correctly.
        systemctl --user restart opentabletdriver
```

**Review by InfinityGhost — APPROVED** [review]

**Review by InfinityGhost — APPROVED** [review]

**Pull request merged**